sourcemash/frontend/static/logotext.svg

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
I think the favicon is worth making the PR for. Did you resize the favicon down to a 16 by 16px image?